cmd/nexctl/main.go

```diff
@@ -24,6 +24,9 @@ const (
 // This variable is set using ldflags at build time. See Makefile for details.
 var Version = "dev"
 
+// Optionally set at build time using ldflags
+var DefaultServiceURL = ""
```

Comment:
Could we set this to the production URL?  Just in case someone builds from source using `go install github.com/nexodus-io/nexodus/cmd/nexctl`

Reply:
done